src/System.Threading.Tasks.Dataflow/src/System.Threading.Tasks.Dataflow.csproj

```diff
@@ -6,7 +6,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>System.Threading.Tasks.Dataflow</RootNamespace>
     <AssemblyName>System.Threading.Tasks.Dataflow</AssemblyName>
-    <AssemblyVersion>4.5.26.0</AssemblyVersion>
+    <AssemblyVersion>4.6.26.0</AssemblyVersion>
```

Comment:
when changing minor version, shouldn't that reset the next number too.
4.5.26.0 -> 4.6.0.0 ?
@ericstj